Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM circleci/node:10.14.1 as builder
+
+WORKDIR /usr/src/app
+
+COPY /src /usr/src/app/src
+COPY /public /usr/src/app/public
+COPY /package.json /usr/src/app/package.json
+COPY /tsconfig.json /usr/src/app/tsconfig.json
+COPY /yarn.lock /usr/src/app/yarn.lock
+
+RUN sudo chmod -R 777 /usr/src/app
+RUN yarn install
+RUN yarn build
+
+FROM nginx:1.16.1
+LABEL maintainer="https://teamdigitale.governo.it"
+
+COPY .env.io-onboarding-pa.development /usr/share/nginx/html/.env.io-onboarding-pa
```

Comment:
instead of copying the development env file in the image, mount the file in the docker-compose conf - this avoids having a development specific Dockerfile that can be reused also for production deployments

Reply:
@cloudify , what if we simply live a generic file only with keys and values not set?